src/Commands/Env/CloneContentCommand.php

```diff
@@ -94,6 +102,15 @@ public function cloneContent(
         }
 
         if (empty($options['files-only'])) {
+            // If the site is a wordpress site, we need to pass the search-replace
+            // option to the API along with the from_url and to_url from each
+            // environment.
+            if ($site->getFramework()->isWordpressFramework()) {
```

Comment:
Does this function include wordpress_network?

Reply:
No idea. Will research on Wednesday. Going to give a development build to the pressies to test.

Reply:
Yes, it checks for both.